imgcodecs.go

```diff
@@ -242,11 +238,18 @@ func IMEncodeWithParams(fileExt FileExt, img Mat, params []int) (buf *NativeByte
 //
 // For further details, please see:
 // https://docs.opencv.org/master/d4/da8/group__imgcodecs.html#ga26a67788faa58ade337f8d28ba0eb19e
-//
 func IMDecode(buf []byte, flags IMReadFlag) (Mat, error) {
 	data, err := toByteArray(buf)
 	if err != nil {
 		return Mat{}, err
 	}
 	return newMat(C.Image_IMDecode(*data, C.int(flags))), nil
 }
+
+func IMDecodeInToMat(buf []byte, flags IMReadFlag, mat Mat) (Mat, error) {
```

Comment:
This function would be better named `IMDecodeIntoMat()` (note case)

Also, it needs to have GoDoc comments with description and a link to the OpenCV function it wraps like all the other functions, please.

Reply:
Seems to me the signature of this function should be:

```go
func IMDecodeInToMat(buf []byte, flags IMReadFlag, dest Mat) error
```

Since it is intended to put the result into `dest` instead of returning a new `Mat`.

Also, the implementation does not do this yet, although it needs to, correct? 😸

Reply:
UPDATE: actually to conform to the function signatures used for other functions it should be

```go
func IMDecodeInToMat(buf []byte, flags IMReadFlag, dest *Mat) error
```